<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="statusBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="callButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Call via Jabber</string>
   </property>
  </widget>
  <widget class="QComboBox" name="agendaDropDown">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectFileButton">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>230</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Agenda File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneNumberEntry">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>211</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
